### PR TITLE
Use CHUNK_SIZE_AIM instead of old environment variables in compose

### PR DESCRIPTION
**Pull request by hardbyte**

### CODE REVIEW

File: tools/docker-compose.yml [hunk 1]
```diff
@@ -68,11 +68,7 @@ services:
       - CELERY_ACKS_LATE=true
       - REDIS_USE_SENTINEL=false
       - CELERYD_MAX_TASKS_PER_CHILD=4096
-      - SMALL_COMPARISON_CHUNK_SIZE=100000000
-      - LARGE_COMPARISON_CHUNK_SIZE=1000000000
-      - SMALL_JOB_SIZE=100000000
-      - LARGE_JOB_SIZE=100000000000
-      - MATCH_THRESHOLD=0.98
```
Comment (gusmith) [inline tools/docker-compose.yml]: Why did you remove the default threshold?

Comment (hardbyte) [reply tools/docker-compose.yml]: Same reason - it isn't a used environment variable so it is just misleading people who might change it expecting something about our software to change...

File: backend/entityservice/settings.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ class Config(object):
     CELERY_ACKS_LATE = os.getenv('CELERY_ACKS_LATE', 'false') == 'true'
 
     # Number of comparisons per chunk (on average).
-    CHUNK_SIZE_AIM = int(os.getenv('CHUNK_SIZE_AIM', '300000000'))
+    CHUNK_SIZE_AIM = int(os.getenv('CHUNK_SIZE_AIM', '300_000_000'))
```
Comment (gusmith) [inline backend/entityservice/settings.py]: I've haven't read the PEP 515 before :) Thanks for the learning.
